software/contrib/update_firmware.py

```diff
@@ -0,0 +1,34 @@
+from machine import bootloader, reset
+from europi_script import EuroPiScript
+from europi import b1, b2, oled
+from os import remove
+
+
+class UpdateFirmware(EuroPiScript):
```

Comment:
[DIscussion] The name 'Update Firmware' feels a little misleading to me since it doesn't actually trigger an update on its own. Something like 'Bootloader' seems more accurate, though perhaps a little less user friendly?

Reply:
Fixed and named it Bootloader, I think that's intuitive enough given how much comes up online thanks to the Pico documentation itself, and we refer to the mode as bootloader mode elsewhere in the EuroPi docs